src/openfermion/utils/_slater_determinants.py

```diff
@@ -19,7 +19,7 @@
 from openfermion.config import EQ_TOLERANCE
 from openfermion.ops import QuadraticHamiltonian
 from openfermion.ops._givens_rotations import (
-        fermionic_gaussian_decomposition, givens_decomposition)
+    fermionic_gaussian_decomposition, givens_decomposition)
```

Comment:
Are you using non-default settings for pycodestyle? Mine does not complain about having an extra four spaces in these situations, and I actually think  the extra space helps with readability sometimes.

Reply:
This is the (optional) PEP8 E126, continuation line over-indented for hanging indent - I prefer only doing the extra spaces when it would otherwise raise E129, visually indented line with same indent as next logical line. I do it with my own PRs (I find it a little harder to read sometimes) but don't mind in other people's, I guess.